docs/data/base/components/click-away-listener/click-away-listener.md

```diff
@@ -40,7 +40,9 @@ You can set the component to listen for **leading events** (the start of a click
 
 {{"demo": "LeadingClickAway.js"}}
 
-> ⚠️ **Note:** when the component is set to listen for leading events, interactions with the scrollbar are ignored.
+:::warning
+⚠️ **Note:** when the component is set to listen for leading events, interactions with the scrollbar are ignored.
+:::
```

Comment:
Is there a convention on the prefix? In this PR we can see how we have been doing it almost (we miss one) all the possible ways before:

- icon + **Note:**
- icon
- Note:
- **Note:**

It would be great to normalize this 😁

Reply:
Yeah, that makes sense! I haven't touched the content of each callout but that'd be great to standardize for sure

Reply:
Good call! 

Now that we have these callouts specifically for "notes" (as opposed to the earlier generic blockquotes) I think it's probably redundant to include the word "Note:" in these. 

I might say the same thing about ⚠️, but I can see it being worthwhile for extra emphasis in `warning` callouts. I also like using 💡 in `info` blocks, but it may not always be applicable. I'm thinking we should either go with icons in all callouts, or else none.